include/mgmt_conn.h

```diff
@@ -108,6 +109,9 @@ int uzfs_get_snap_zv_ionum(zvol_info_t *, uint64_t, zvol_state_t **);
 int uzfs_zvol_get_snap_dataset_with_io(zvol_info_t *zinfo,
     char *snapname, uint64_t *snapshot_io_num, zvol_state_t **snap_zv);
 
+int disable_zinfo_conn(char *zv_name);
```

Comment:
disable_zinfo_mgmt_conn and enable_zinfo_mgmt_conn

Reply:
done